admin/partials/helpers/fields/yikes-mailchimp-select-field.php

```diff
@@ -18,6 +18,7 @@
 	<!-- Dropdown -->
 	<select class="custom-select-field" name="custom-field[<?php echo $field['id']; ?>]">
 		<?php foreach( $field['options'] as $value => $label ) { ?>
+			<option value="" disabled selected><?php echo isset( $field['placeholder'] ) ? $field['placeholder'] : 'Select...'; ?></option>
```

Comment:
@chrisvo3 move this outside of the foreach loop because I'm not sure why its working for us locally but it should be before the foreach loop to make sure its never duplicated.